src/components/Menu/index.tsx

```diff
@@ -127,6 +127,11 @@ const MenuItem = styled(ExternalLink)`
     cursor: pointer;
     text-decoration: none;
   }
+  :focus-visible {
+    border-radius: 0.5rem;
+    text-decoration: none;
+    outline: 1px solid ${({ theme }) => theme.bg3};
+  }
 `
 
 const InternalMenuItem = styled(Link)`
```

Comment:
Should this use StyledInternalLink? If so, the :focus-visible be on that component, so it correctly displays for all internal links.

Reply:
Yes I tried there, but the outline style is not matching for all links, its good to customize the focus-visible on demand!